site/en/docs/extensions/mv3/screen_capture/index.md

```diff
@@ -0,0 +1,155 @@
+---
+layout: "layouts/doc-post.njk"
+title: "Audio recording and screen capture"
+seoTitle: "Chrome Extensions: Audio recording and screen capture"
+date: 2023-04-14
+updated: 2023-04-14
+description: How to record audio or video from a tab, window or screen
+---
+
+This guide explains different approaches for recording audio and video from a tab, window or
+screen using APIs such as [chrome.tabCapture][tabcapture] or [getDisplayMedia][get-display-media].
+
+## Common Use Cases
+
+### Screen Recording {: #screen-recording}
+
+For screen recording, use the [getDisplayMedia][get-display-media] API. This provides the user with
+the ability to select which tab, window or screen they wish to share and provides a clear indication
+that recording is taking place.
+
+{% Img src="image/wVNVUJS8Z8O04i1tJKSdsp6nkRQ2/Q78okJG9DlfNwzyf2mly.png", alt="Screen share dialog for example.com", width="728", height="610" %}
+
+```js
+// User will be prompted to select what they would like to share.
+const stream = await navigator.mediaDevices.getDisplayMedia({ audio: true, video: true });
```

Comment:
```suggestion
The following example requests access to record both audio and video.
```js
const stream = await navigator.mediaDevices.getDisplayMedia({ audio: true, video: true });
```

Reply:
Removing 'will be prompted'

The problem is that the order of your presentation the order of what you're describing are moving in opposite directions. This code triggers the dialog box (the future), but you show the dialog box in the reader's past. The text I suggested above replaces your comment and corrects the order. 

The other option is to swap this code with the proceeding image. The advantage of turning this into an example is that the reader gets to absorb what `getDisplayMedia()` does generally before giving them different ways to use it. I admit that this level of splitting may be overkill for something this simple, but it's a useful trick to learn.

Reply:
That makes sense! I think this sort of thing is hard to see when you know what's coming in the article so appreciate you pointing it out.